packages/components/src/components/Cell/NavigationCell.tsx

```diff
@@ -61,24 +68,39 @@ export const NavigationCell = ({
             {...cellProps}
         >
             <View style={styles.contentContainer}>
-                <Typography
-                    group="cell"
-                    variant="title"
-                    numberOfLines={1}
-                    color={disabled ? "textDisabled" : undefined}
-                >
-                    {title}
-                </Typography>
-                {description && (
+                <View style={styles.titleDescriptionContainer}>
                     <Typography
                         group="cell"
-                        variant="description"
-                        numberOfLines={2}
-                        color={disabled ? "textDisabled" : "textTertiary"}
+                        variant="title"
+                        numberOfLines={1}
+                        color={disabled ? "textDisabled" : undefined}
                     >
-                        {description}
+                        {title}
                     </Typography>
-                )}
+                    {description && (
+                        <Typography
+                            group="cell"
+                            variant="description"
+                            numberOfLines={2}
+                            color={disabled ? "textDisabled" : "textTertiary"}
+                        >
+                            {description}
+                        </Typography>
+                    )}
+                </View>
+                {title &&
```

Comment:
On mobile, this looks a little strange:
<img width="474" alt="Screenshot 2023-12-18 at 07 27 03" src="https://github.com/equinor/mad/assets/36899138/679effaf-a802-4d4d-bb4c-6d47d3519cc7">

From the designs that i suspect that this change comes from, it looks like this on iPhone vs iPad:
<img width="550" alt="Screenshot 2023-12-18 at 07 30 57" src="https://github.com/equinor/mad/assets/36899138/47982f08-a049-4e61-9f85-7a018cbc4512">


I suggest we try to style with these rules:

- On the main title line, the titles will be spread out as far away from each other as possible (i.e `justifyContent: "space-between"`
- If any title (normal or additional) comes into contact with any other, we move at least one additional title to the bottom of the cell

<img width="550" alt="Screenshot 2023-12-18 at 07 41 06" src="https://github.com/equinor/mad/assets/36899138/5581669e-c65d-4773-bbf8-4505968cd9e3">


Any thoughts on this @EmilieOlsen ?

Reply:
![image](https://github.com/equinor/mad/assets/111871753/78d9bc99-434e-4aee-9f2c-2157207e6ced)

**For mobile:** 
The NormalTitle can be long, so there will probably not be any more room for AdditionalTitle, so this should go under the Description

**iPad:**
Another solution could be to use similar layout as mobile. AdditionalTitles under Description. Attached iPad layout is somewhat dependent on the length on the NormalTitle.

Reply:
Last point is interesting, yes. It could be that the iPad title is long enough to cause issues as well. Either we add ellipsises to it in that case ("..."), or we have it automatically move the additional titles to the bottom as you mention.
If we want all cells to be consistent (i.e not have just one of them move the additional titles to the bottom), I think we should just cut off the main title to allow for the additional titles.

Reply:
Sounds good to me!